### ISSUE TITLE
New name of this plugin??

### ISSUE THREAD

**Issue by RyotaUshio** [RyotaUshio/obsidian-latex-theorem-equation-referencer#210]
**TL;DR: I'm looking for a nice new name for this plugin.**

I started writing Math Booster right after I started using Obsidian, coming from Overleaf. It was because Obsidian was overall too painful to do math-ish stuff in but at the same time I couldn't dismiss its possibilities.

With this motivation in mind, I at first tried to include **everything** I thought Obsidian lacked to provide a comfortable environment for doing math. This made this plugin a kind of jack-of-all-trades, and this is reflected in the ambiguous name _Math Booster_ or the phrase [_turn your Obisidian into LaTeX on steroids_](https://github.com/RyotaUshio/obsidian-math-booster/blob/52992317f7acc596e5bbf7cdb7dcf33ae8f1a37d/manifest.json#L6) --- what the hell do they mean??

Recently, I decided to make this plugin more focused around indexing & referencing of theorems and equations.

> Math Booster is an [Obsidian.md](https://obsidian.md/) plugin that provides a powerful indexing & referencing system for theorems & equations in your vault, bringing $\LaTeX$-like workflow into Obsidian.

And I also plan to move some features irrelevant to this theme to another separate plugin (https://github.com/RyotaUshio/obsidian-math-in-callout).

So now we need a new, more concrete, self-descriptive name for this plugin. I appreciate it if anyone could share their ideas.

**Comment by RyotaUshio** [comment]
@mayurankv Thanks for the suggestion! That's dead simple, very good. I might want to add "equation" to the title.

I'm still not very sure whether the name naturally implies the automatic equation numbering feature.
Do you feel it's natural that it is part of a plugin called "Theorem & Equation Referencer"?

**Comment by mayurankv** [comment]
Yes that would actually be perfect! (I do recall being very confused by the name math booster aha). Keeping latex in the Id would be useful as well but otherwise I think that's good.

**Comment by Flesko** [comment]
I agree that something with `reference` and `latex` should be in the name. I came across the plugin completely by chance, although I had already searched for something similar a few times beforehand.

**Comment by mayurankv** [comment]
I think that is good!

**Comment by RyotaUshio** [comment]
Hi all, just want to let you know Math Booster has been renamed to **LaTeX-like Theorem & Equation Referencer** since 2.2.0.

While the community plugin browser hasn't reflect the new name, it will be updated accordingly once approved by the Obsidian team. https://github.com/obsidianmd/obsidian-releases/pull/2769

You guys helped me a lot, thank you!
